pkg/config/config.go

```diff
@@ -362,6 +362,10 @@ func (b *Blockstore) IsBackwardsCompatible() bool {
 	return false
 }
 
+func (b *Blockstore) SigningKey() SecureString {
```

Comment:
A hell of a nit, but -
Better to place this in line 242, right after the rest of the `StorageConfig` interface implementation.